pyoverkiz/client.py

```diff
@@ -137,6 +128,14 @@ async def __aexit__(
     ) -> None:
         await self.close()
 
+    # pylint: disable=unused-argument
+    async def _relogin(self, *args: Any, **kwargs: Any) -> None:
+        await self.login()
```

Comment:
I would be surprised if that works. `backoff` only passes a dict with some invocation details to the `on_backoff` handler. Thus `self` would be missing.

The old solution with `invocation["args"][0]` might be ugly, but it should work.

Reply:
I will double check this before merging, good comment.